src/window.py

```diff
@@ -343,7 +343,7 @@ def countdown():
             i = 5
             while i != 0:
                 print('Reloading configuration in ' + str(i) + ' seconds...')
-                GLib.idle_add(lambda: self.titlebar_title.set_title(_('Reloading last connections in ') + str(i) + _(' seconds...')))
+                GLib.idle_add(lambda: self.titlebar_title.set_title(_('Reloading last connections in {0} seconds...'.format(i))))
```

Comment:
@mijorus Is this usage correct? I'm not a python developer and I just wanted to fix split sentence problem?

More information: https://wiki.gnome.org/TranslationProject/DevGuidelines/Never%20split%20sentences